packages/statemanager/tests/stateManager.spec.ts

```diff
@@ -1,22 +1,22 @@
-import * as tape from 'tape'
-import { keccak256 } from 'ethereum-cryptography/keccak'
-import { bytesToHex } from 'ethereum-cryptography/utils'
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
 import {
   Account,
   Address,
-  toBuffer,
   KECCAK256_RLP,
   KECCAK256_RLP_S,
+  toBuffer,
   unpadBuffer,
   zeros,
 } from '@ethereumjs/util'
-import { Chain, Common, Hardfork } from '@ethereumjs/common'
-import { DefaultStateManager } from '../src'
-import { createAccount } from './util'
-
+import { keccak256 } from 'ethereum-cryptography/keccak'
+import { bytesToHex } from 'ethereum-cryptography/utils'
+import * as tape from 'tape'
 // explicitly import `inherits` to fix karma-typescript issue
 import { inherits } from 'util' // eslint-disable-line
```

Comment:
Not part of your work either, but could we figure out which eslint rule needs to be disabled specifically on this line, instead of disabling all linting?